modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -239,6 +242,15 @@ conf = {{{packageName}}}.Configuration(
            Default values is 100, None means no-limit.
         """
         {{/asyncio}}
+        {{^asyncio}}
+        self.connection_pool_maxsize = multiprocessing.cpu_count() * 5
```

Comment:
Before #16802, the code was 

```py
        if maxsize is None:
            if configuration.connection_pool_maxsize is not None:
                maxsize = configuration.connection_pool_maxsize
            else:
                maxsize = 4
```

So it was set to `4` only if `configuration.connection_pool_maxsize` was explicitly set to `None`, instead of letting it default to `multiprocessing.cpu_count() * 5`. I'm reverting `configuration.mustache` to the same code it had before #16802.

From how I understand this setting in urllib3, you should not get more connections than you have threads. This is only an upper limit.

TBH, this fix is most convenient for our use-case (I think). But it would actually be cleanest for the OpenAPI generator to not set any custom default, go with urllib3's default, and make it configurable. What do you think?

Reply:
Ah yes, you are right, I just re-read #16802: so `multiprocessing.cpu_count() * 5` was indeed most probably the _most_ default value.

Then:

* Let's roll back to the previous value `multiprocessing.cpu_count() * 5` to restore the previous behavior.
* It's still changeable once the `Configuration` object has been created, so it can be adjusted on a per-client basis.

Reply:
Great, thanks! Then the current state of the PR is good and we can resolve?